Dockerfile

```diff
@@ -1,3 +1,22 @@
+
+###################################
+#Build stage
+FROM golang:1.9-alpine3.6 AS build-env
+
+ARG GITEA_VERSION
+ARG TAGS="bindata sqlite"
+
+#Build deps
+RUN apk --no-cache add git build-base
+
+#Setup repo
+ADD . ${GOPATH}/src/code.gitea.io/gitea
+WORKDIR ${GOPATH}/src/code.gitea.io/gitea
+
+#Checkout version if set
+RUN if [ -n "${GITEA_VERSION}" ]; then git checkout ${GITEA_VERSION}; fi \
```

Comment:
AFAIU, when you don't write `RUN set -eu; <some other commands>`, each run will exit without error. The build will fail anyways of course, but stdout will be polluted with more errors due to each subsequent command printing fails instead of first one. Not a blocker.